src/processes/services/AuthService.ts

```diff
@@ -9,8 +9,7 @@ export default class AuthService {
   }
 
   static async signup(newUser:ISignUpData): Promise<AxiosResponse<AuthResponse>> {
-    console.log(newUser)
-    return $api.post<AuthResponse>('/signup', {...newUser })
+     return $api.post<AuthResponse>('/signup', {...newUser })
```

Comment:
```suggestion
     return $api.post<AuthResponse>('/signup', newUser )
```
здесь можно не клонировать, я думаю